go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/lightninglabs/pool v0.6.2-beta.0.20230329135228-c3bffb52df3a
 	github.com/lightninglabs/pool/auctioneerrpc v1.1.0
 	github.com/lightninglabs/protobuf-hex-display v1.4.3-hex-display
-	github.com/lightningnetwork/lnd v0.16.0-beta
+	github.com/lightningnetwork/lnd v0.16.1-beta.rc1
```

Comment:
think we wanna wait till 0.16.1-beta before doing a release here. 

i'm also happy to do the LiTd release btw 👍